### PR TITLE
op-guide: add cross DC deployment guide

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### CODE REVIEW

File: op-guide/cross-dc-deployment.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Cross-DC Deployment Solutions
+category: deployment
+---
+
+# Cross-DC Deployment Solutions
+
+As a NewSQL database, TiDB excels in the best features of the traditional relational database and the scalability of the NoSQL database and is of course, highly available across data centers (hereinafter referred to as DC). This document is to introduce different deployment solutions in cross-DC environment.
```
Comment (disksing) [inline op-guide/cross-dc-deployment.md]: :exclamation:

File: op-guide/cross-dc-deployment.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## 3-DC Deployment Solution
+
+TiDB, TiKV and PD are distributed among 3 DCs, which is the most common deployment solution which the highest avaiability.
```
Comment (disksing) [inline op-guide/cross-dc-deployment.md]: :exclamation:

Comment (kennytm) [inline op-guide/cross-dc-deployment.md]:
```suggestion
TiDB, TiKV and PD are distributed among 3 DCs, which is the most common deployment solution with the highest availability.
```

**Comment by disksing** [comment]
@queenypingcap I have marked the difference places with the original article with :exclamation:

**Pull request merged**